content/docs/command-reference/push.md

```diff
@@ -57,6 +57,8 @@ Note that the `dvc status -c` command can list files tracked by DVC that are new
 in the cache (compared to the default remote.) It can be used to see what files
 `dvc push` would upload.
 
+> Note that `dvc push` doesn't upload `dvc import` data.
```

Comment:
not related to this PR

also, seems to be in a place that no one would ever notice this? if we care about search - we can put in the `dvc import`?

Reply:
Rolled back.

Reply:
> we can put in the dvc import

BTW this note is already found in `import`.

Reply:
Also, that note had some errors which I had fixed in this PR. But I rolled back the fix for now, as it may be unrelated.